lxd/instance/instance_interface.go

```diff
@@ -96,6 +96,10 @@ type Instance interface {
 	Rename(newName string, applyTemplateTrigger bool) error
 	Update(newConfig db.InstanceArgs, userRequested bool) error
 
+	// EFI vars handling.
```

Comment:
We have a VM specific interface these should be added to:

https://github.com/canonical/lxd/blob/main/lxd/instance/instance_interface.go#L186-L190

Then where you need to access them you can type assert e.g.

```
inst.(instance.VM).AgentCertificate()
```

Reply:
Fixed, thanks!